## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 452340`
Commit: [`b8073a577ad72fc8e9cc52283cc5ac8cae0c95c7`](https://github.com/NixOS/nixpkgs/commit/b8073a577ad72fc8e9cc52283cc5ac8cae0c95c7) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/b8073a577ad72fc8e9cc52283cc5ac8cae0c95c7..pull/452340/head))
Merge: [`2a41843c5cf63d6034622eae360c98dafd09dc90`](https://github.com/NixOS/nixpkgs/commit/2a41843c5cf63d6034622eae360c98dafd09dc90)

Logs: https://github.com/Sigmanificient/nixpkgs-review-gha/actions/runs/18554596565


---
### `x86_64-linux`
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python312Packages.scalene</li>
    <li>python312Packages.scalene.dist</li>
    <li>scalene (python313Packages.scalene)</li>
    <li>scalene.dist (python313Packages.scalene.dist)</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python312Packages.scalene</summary>
<pre>                errno_num = int(hex_errno, 16)
                if err_msg == &quot;noexec:chdir&quot;:
                    err_msg = &quot;&quot;
                    # The error must be from chdir(cwd).
                    err_filename = cwd
                elif err_msg == &quot;noexec&quot;:
                    err_msg = &quot;&quot;
                    err_filename = None
                else:
                    err_filename = orig_executable
                if errno_num != 0:
                    err_msg = os.strerror(errno_num)
                if err_filename is not None:
&gt;                   raise child_exception_type(errno_num, err_msg, err_filename)
E                   FileNotFoundError: [Errno 2] No such file or directory: &#x27;/build/scalenevkw9ljsh/python&#x27;

/nix/store/27fr7ynpilzb7rzmay73g905jbl8br4a-python3-3.12.11/lib/python3.12/subprocess.py:1955: FileNotFoundError
=========================== short test summary info ============================
FAILED tests/test_nested_package_relative_import.py::test_nested_package_relative_import - FileNotFoundError: [Errno 2] No such file or directory: &#x27;/build/scalenevkw9ljsh/python&#x27;
================= 1 failed, 154 passed, 1 deselected in 29.73s =================</pre>
</details>
<details>
<summary>scalene</summary>
<pre>                errno_num = int(hex_errno, 16)
                if err_msg == &quot;noexec:chdir&quot;:
                    err_msg = &quot;&quot;
                    # The error must be from chdir(cwd).
                    err_filename = cwd
                elif err_msg == &quot;noexec&quot;:
                    err_msg = &quot;&quot;
                    err_filename = None
                else:
                    err_filename = orig_executable
                if errno_num != 0:
                    err_msg = os.strerror(errno_num)
                if err_filename is not None:
&gt;                   raise child_exception_type(errno_num, err_msg, err_filename)
E                   FileNotFoundError: [Errno 2] No such file or directory: &#x27;/build/scalene5f62nqo1/python&#x27;

/nix/store/62fdlzq1x1ak2lsxp4ij7ip5k9nia3hc-python3-3.13.7/lib/python3.13/subprocess.py:1972: FileNotFoundError
=========================== short test summary info ============================
FAILED tests/test_nested_package_relative_import.py::test_nested_package_relative_import - FileNotFoundError: [Errno 2] No such file or directory: &#x27;/build/scalene5f62nqo1/python&#x27;
================= 1 failed, 154 passed, 1 deselected in 29.96s =================</pre>
</details>
</details>


---
### `aarch64-linux`
:white_check_mark: *No rebuilds*

--- a/pkgs/development/python-modules/scalene/default.nix
+++ b/pkgs/development/python-modules/scalene/default.nix
@@ -38,7 +38,7 @@ in
 
 buildPythonPackage rec {
   pname = "scalene";
-  version = "1.5.52";
+  version = "1.5.55";
   pyproject = true;
   disabled = pythonOlder "3.9";
 
@@ -46,7 +46,7 @@ buildPythonPackage rec {
     owner = "plasma-umass";
     repo = "scalene";
     tag = "v${version}";
-    hash = "sha256-8WE/tR0tGwdNSPtieS90QAOFlS66h/JxaV2LvpZjx2E=";
+    hash = "sha256-aO7l/paYqbneDArAbXxptIlMGfvc1dAaFLucEj/7xbk=";
   };
 
   patches = [
@@ -95,6 +95,8 @@ buildPythonPackage rec {
   disabledTests = [
     # Flaky -- socket collision
     "test_show_browser"
+    # File not found
+    "test_nested_package_relative_import"
   ];
 
   # remove scalene directory to prevent pytest import confusion

